src/Agent.Worker/Handlers/StepHost.cs

```diff
@@ -97,12 +98,15 @@ public string ResolvePathForStepHost(string path)
             // remove double quotes around the path
             path = path.Trim('\"');
 
+            // if running on OS X, just translate the path
+            if (PlatformUtil.RunningOnMacOS)
+            {
+                return Container.TranslateToContainerPath(path);
```

Comment:
Why do we only need this on mac?

Reply:
we may need it on windows too. We need this when the OS in the container doesn't match the OS on the host. I am going to make another pass through this now that I understand the problem a little better and we have a use case for running linux containers on windows as well.